src/test/java/org/folio/migration/FeeFineTypesMigrationTest.java

```diff
@@ -0,0 +1,70 @@
+package org.folio.migration;
+
+import static org.folio.rest.utils.ResourceClients.tenantClient;
+import static org.folio.test.support.matcher.FeeFineMatchers.hasAllAutomaticFeeFineTypes;
+
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.tools.PomReader;
+import org.folio.test.support.ApiTests;
+import org.junit.Test;
+
+import io.vertx.core.Vertx;
+
+public class FeeFineTypesMigrationTest extends ApiTests {
```

Comment:
The script doesn't appear to be a regular upgrade script, more like the loading of default reference data.

Maybe it is worth naming this test after that purpose instead.

```suggestion
public class FeeFineTypesDefaultReferenceRecordsTest extends ApiTests {
```

Reply:
Make sense, renamed.